cryptlex/lexactivator/lexactivator.py

```diff
@@ -165,6 +165,26 @@ def SetDebugMode(enable):
         status = LexActivatorNative.SetDebugMode(enable)
         if LexStatusCodes.LA_OK != status:
             raise LexActivatorException(status)
+        
+    @staticmethod
+    def SetCacheMode(enable: bool):
+        """Enables or disables in-memory caching for LexActivator.
+        Enables or disables in-memory caching for LexActivator. This function is designed to control caching
+        behavior to suit specific application requirements. Caching is enabled by default to enhance performance.
+        Disabling caching is recommended in environments where multiple processes access the same license on a 
+        single machine and require real-time updates to the license state.
+
```

Comment:
@muneebkq Please refer to the docs of other functions and update the SetCacheMode() docs.